editor/src/components/inspector/sections/component-section/data-picker-utils.tsx

```diff
@@ -156,3 +157,38 @@ function outletNameHack(
   const component = findContainingComponentForPathInProjectContents(target, projectContents)
   return component?.name ?? namePossiblyOutlet
 }
+
+export function cartoucheFolderOrInfo(
```

Comment:
this is only used by a helper function of the top bar input field, but the top bar input field no longer cares about cartouche role, so this should be deleted instead of turned into an exported helper function.